configuration/configuration.go

```diff
@@ -391,6 +401,28 @@ func (gc *GlobalConfiguration) initACMEProvider() {
 	}
 }
 
+func getSafeACMECAServer(caServerSrc string) string {
+	if len(caServerSrc) == 0 {
+		return DefaultAcmeCAServer
+	}
+
+	if strings.HasPrefix(caServerSrc, "https://acme-v01.api.letsencrypt.org") {
+		caServer := strings.Replace(caServerSrc, "v01", "v02", 1)
+		log.Warnf("The CA server %q refer to a v01 endpoint of the ACME API, please change to %q.", caServerSrc, caServer)
```

Comment:
s/refer/refer**s**
No sure users will understand that a modification will be automatically done.
WDYT to use a more explicit message?